samples/remote_attestation/common/attestation.cpp

```diff
@@ -73,17 +73,17 @@ bool Attestation::generate_remote_evidence(
  * the claim name, claim list, and its size, returns the value stored with that
  * claim name in the list.
  */
-static const oe_claim_t* _find_claim(
+static const void* _find_claim(
```

Comment:
Please return oe_claim_t* so caller knows the size of the value. It's a security concern when you blindly read/write out of memory boundary.